package.json

```diff
@@ -68,70 +68,79 @@
       "transform-inline-environment-variables"
     ],
     "presets": [
-      "es2017",
-      "es2015",
+      "env",
       "react"
     ],
     "env": {
       "test": {
         "plugins": [
           "transform-es2015-modules-commonjs"
         ]
+      },
+      "development": {
+        "plugins": [
+          "react-hot-loader/babel"
+        ]
       }
     }
   },
   "devDependencies": {
-    "babel-cli": "^6.24.0",
-    "babel-core": "^6.24.0",
+    "@k-redux-router/core": "^1.0.0",
+    "@k-redux-router/react-redux": "^1.0.0",
+    "babel-cli": "^6.26.0",
```

Comment:
You can do it in an other PR after this one, but babel is now "7", this is new package organisation with "@babel/core", "@babel/cli", "@babel/plugin-transform-bla", etc

This is a lot of changes.
You can get example of migration there: https://github.com/alakarteio/k-ramel/commit/e23cbff48fac387ad37fd4f29fd08f64f3f5bc19

But once again, you should wait this PR to be merged.